testData/solutions/linux.json5

```diff
@@ -1289,34 +1289,122 @@
                     "path": "net.gpii.uioPlus"
                 },
                 "supportedSettings": {
-                    "lineSpace": {},
-                    "fontSize": {},
-                    "characterSpace": {},
-                    "inputsLargerEnabled": {},
-                    "contrastTheme": {},
-                    "selfVoicingEnabled": {},
-                    "selectionTheme": {},
-                    "tableOfContentsEnabled": {},
-                    "simplifiedUiEnabled": {},
-                    "syllabificationEnabled": {},
-                    "captionsEnabled": {}
-                },
-                "capabilitiesTransformations": {
-                    "lineSpace": "http://registry\\.gpii\\.net/common/lineSpace",
+                    "captionsEnabled": {
+                        "schema": {
+                            "title": "Captions",
+                            "description": "Request videos to display captions.",
+                            "type": "boolean",
+                            "default": false
+                        }
+                    },
+                    "characterSpace": {
+                        "schema": {
+                            "title": "Character Space",
+                            "description": "Adjust the space between characters.",
+                            "type": "number",
+                            "default": 1,
+                            "minimum": 0.9,
+                            "maximum": 2,
+                            "multipleOf": 0.1
+                        }
+                    },
+                    "contrastTheme": {
+                        "schema": {
+                            "title": "Contrast Theme",
+                            "description": "Change text and background colors.",
+                            "type": "string",
+                            "default": "default",
+                            "enum": [
+                                "default",
+                                "bw",
+                                "wb",
+                                "by",
+                                "yb",
+                                "lgdg",
+                                "gw",
+                                "bbr"
+                            ]
+                        }
```

Comment:
This won't be a valid schema until you add ["enumLabels"](https://github.com/the-t-in-rtf/gpii-json-schema/blob/GPII-3083/docs/gss.md#the-enumlabels-errors-and-hint-attributes), one for each option.

Long term raw strings will be moved to a message bundle, but I wouldn't expect you to do that in this pull.  Just enter the raw strings for each label and I'll move it later on.